.circleci/config.yml

```diff
@@ -43,3 +43,28 @@ jobs:
       - run:
           name: Test installing Mesh without CGO
           command: CGO_ENABLED=0 go install ./...
+  build-ts-client:
```

Comment:
We should start calling this a TypeScript _RPC_ client in order to reduce confusion. In the near future we will also have TypeScript bindings for running and interacting with Mesh in a browser.

How about `build-and-test-ts-rpc-client`? Alternatively we could separate this into two tasks `build-ts-rpc-client` and `test-ts-rpc-client`.

Reply:
Went with `build-and-test-ts-rpc-client`